docsrc/imap/reference/manpages/systemcommands/cyr_expire.rst

```diff
@@ -160,6 +160,11 @@ Options
     Remove any user flags which are not used by remaining (not expunged)
     messages.
 
+.. option:: -b num, --batchsize=num
```

Comment:
In the synopsys the parameter is called batchsize.

Reply:
I have changes this to match, in the same style as the others.